include/votca/csg/io/dlpolytopologyreader.h

```diff
@@ -0,0 +1,471 @@
+/*
```

Comment:
How many different topologies do we really need?

Reply:
I am not sure the whole templating is necessary here, we only have the csg_topology